Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -100,18 +100,33 @@ LogManager::LogManager()
   IniFile::Section* options = ini.GetOrCreateSection("Options");
   bool write_file;
   bool write_console;
+  bool write_window;
   options->Get("WriteToFile", &write_file, false);
   options->Get("WriteToConsole", &write_console, true);
+  options->Get("WriteToWindow", &write_window, true);
+
+  // Set up log listeners
+  int verbosity;
+  options->Get("Verbosity", &verbosity, 0);
+
+  // Ensure the verbosity level is valid
+  if (verbosity < 1)
```

Comment:
I can't recall, do we have access to c++ 17 features?  If so, we could leverage std::clamp